javascript-data-model/order-history.js

```diff
@@ -0,0 +1,73 @@
+var orderHistory = [
+  [
+    {
+      orderPlaced: 'August 4, 2020',
+      orderTotal: 34.00,
+      shipTo: 'JS Masher',
+      delivered: 'Aug 8, 2020'
+    },
+    {
+      itemName: 'JavaScript for impatient programmers',
+      itemCost: 31.55,
+      returnWindowClosed: 'Sep 7, 2020',
+      otherItemInfo: {
+        author: 'Rauschmayer, Dr. Axel'
+      }
+    }
```

Comment:
Instead of storing this in a sibling object in this array, you should store data on items ordered in a property on the object above that holds data on the order itself.